test/extended/builds/cluster_config.go

```diff
@@ -114,6 +115,38 @@ var _ = g.Describe("[sig-builds][Feature:Builds][Serial][Slow][Disruptive] alter
 			o.Expect(ok).To(o.BeTrue())
 			return build
 		}
+		checkPodResource = func(containers []v1.Container, resources configv1.BuildDefaults) {
+			defaultResources := resources.Resources
+			o.Expect(containers).NotTo(o.BeNil())
+			foundLimitCpu := false
+			foundLimitMem := false
+			foundRequestCpu := false
+			foundRequestMem := false
+			for _, container := range containers {
+				o.Expect(container.Resources).NotTo(o.BeNil())
```

Comment:
I would add an `o.By()` call here so we know which container doesn't have the right spec:

```
o.By(fmt.Sprintf("checking resources of container %s", container.Name))
```

Reply:
Yes, it's more clear to add this. Thanks!  @adambkaplan